guide/installation-and-configuration/general-installation/installation-enterprise.markdown

```diff
@@ -94,10 +94,18 @@ The disk that serves PostgreSQL (/var/cfengine/state/pg) should be able to perfo
 
 **Note** Your storage IOPS specification may be given in 4KiB block size, in which case you would need to divide it by 4 to get the corresponding 16KiB *theoretical maximum*.
 
+### Open file descriptors
+
+The policy server should ideally be able to accept connections from all clients; i.e. to allow at least as many incoming connections as there are clients.
```

Comment:
Should I add a line after this saying, roughly, "This isn't essential: the system should work acceptably without it, but it may work significantly better with it." ?